docs/architecture/db-scheme.md

```diff
@@ -0,0 +1,8 @@
+# Database Scheme
+
+Diagram below represents database scheme. 
+Fields marked with the red color are planned for upgrade:
+- `runtimes.auth` field will be changed to `JSONB`
```

Comment:
When will we update the schema regarding the runtimes? Please create an issue with this change